.github/workflows/build.yml

```diff
@@ -189,7 +195,62 @@ jobs:
         SSH_USER: mixxx
         UPLOAD_ID: ${{ github.run_id }}
 
+    - run: |
+        if ! gh release view --repo "${{ github.repository }}" "${RELEASE_NAME}" &> /dev/null
+        then 
+          gh release create --repo "${{ github.repository }}" "${RELEASE_NAME}" \
+            --generate-notes \
+            --latest \
+            --target "${{ github.push.after }}" ${{ !endsWith( github.event.ref, '-rel' ) && '--prerelease' || '' }}
+        fi
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        RELEASE_NAME: ${{ env.MIXXX_VERSION }}-${{ steps.vars.outputs.sha_short }}
+      if: ${{ github.event_name == 'push' }}
+      id: ghrelease
+      shell: bash
+      name: "Creates a release in GitHub if it doesn't exist yet"
```

Comment:
We normally have the name on top. You seems to have reverted the sort order of elements. 
Please keep the normal sort order.

Reply:
> normal sort order

Does that mean just "name first" or is there some kind of bespoke ranking I should follow?

Reply:
I am not sure if there is something like this. 
It is just a good practice to follow the examples in the GitHub Docs: 
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions

```
- name:
  if:
  env:
  run: 
```

Reply:
I have no strong demands in detail, but it shall be consistent with our other workflow for a good reading experience.   

In https://github.com/mixxxdj/mixxx/blob/main/.github/workflows/build.yml we have: 

* name
* id
* if
* uses 
* with
* shell
* run
* working-directory
* env (above if when required for if)